tests/test_utils.py

```diff
@@ -113,6 +113,8 @@ def _test_specs(job, expected_job_id, expected_endpoint):
         _test_specs('default/2/3', '1/2/3', 'default')
         _test_specs('prod/2/3', '2/2/3', 'default')
         _test_specs('vagrant/2/3', '3/2/3', 'vagrant')
+        _test_specs('https://dash.scrapinghub.com/p/7389/job/259/1/#/log/line/0',
```

Comment:
Could you shorten this line to 79 characters? I'm a little pedantic about PEP8, sorry :D